config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
git rm --cached config/database.yml
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

Reply:
does it mean that copy all the coding from the database.yml to the .gitignore?
after this run that code?